### DISCUSSION TITLE
Hero 11 Sync AND Jam Sync Timecode always set unwanted Start Time

### DISCUSSION THREAD

**Discussion by 45robertf** [gopro/labs#1121] [General]
Every time I use a QR code to sync the time on my Hero 11, I have to press the power button to CANCEL TIME DELAY. This occurs whether I sync using Jam Sync Timecode under the Camera Mode or SYNC under the Extras. I have replaced the firmware (v2.30 and labs firmware (v2.30.70). Scheduled Capture is OFF under the the camera settings. I do not want a time delay and have no idea how I might have set one up. Google and YouTube searches have fruitless.

![20241010_110818 1](https://github.com/user-attachments/assets/2904e09c-ef02-4fdd-9957-2809d15f579b)
![20241010_110917 1](https://github.com/user-attachments/assets/dd924781-f5c8-41bd-a6c3-156ad211a77b)

This is never an issue on my Hero 12.

**Comment by dnewman-gpsw** [comment]
Share the qr code that has the error.   Also remove any delay event you have (delete files MISC/delay.txt and MISC/qr.bin.)  This not typical, so it is something in your setup.

**Comment by 45robertf** [comment]
![Screenshot_20241010_140931](https://github.com/user-attachments/assets/57bf2648-2148-4973-b4c7-24af85166522)

It turns out that many of the features in the QR Code app on my Galaxy S23 result in the same error, and it is happening on both the H11B and H12B. If I open up https://gopro.github.io/labs/control/ on my laptop and read the QR code with either camera, it appears to work correctly. I uninstalled then reinstalled the QR Code app on the S23 with no improvement. I pasted a screenshot of the JAM SYNC TIMECODE mode above. I took this screenshot after I reinstalled the app. I can choose other modes with the same result, although at times it was putting the cameras into the motion trigger mode instead of the delay mode.

I suppose I need to clean out some other damaged files on my phone, but I have no idea what files or what path they are under. Furthermore, my previous search for the MISC/delay.txt and MISC/qr.bin files failed to locate them. Please advise.

Thanks for your support,
Robert

**Reply by dnewman-gpsw** [reply]
User error.  It is doing exactly what you asked.  Pressure Clear, start again, and only use one command per QR code, well until you learn what is in each command.

![image](https://github.com/user-attachments/assets/9ee07167-1046-41ea-9d45-9ddf1468d194)

**Reply by dnewman-gpsw** [reply]
Also never set !Mwxyz and use a !R (repeat), could bad for your camera.
![image](https://github.com/user-attachments/assets/75ce5c7c-d4a9-4d38-ac0f-50ce7d7088c0)

**Comment by 45robertf** [comment]
None of those commands were intentional. I imagine that the "Motion 5" command got in there from previous experimentation with motion trigger. That was months ago. I don't understand why any of these commands were associated with the Jam Sync Timecode mode.

Deleting the "Motion 5" command and pressing clear restored functionality.

Thanks for your help. I wasted hours attempting to solve this before you solved it in seconds.
